packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/byoc/index.ts

```diff
@@ -1,15 +1,21 @@
 import * as FEAAS from '@sitecore-feaas/clientside/react';
+import * as Events from '@sitecore-cloudsdk/events/browser';
 import '@sitecore/components/context';
 import dynamic from 'next/dynamic';
-import { context } from 'lib/context';
+import config from 'temp/config';
 /**
  * This is an out-of-box bundler for External components (BYOC) (see Sitecore documentation for more details)
  * It enables registering components in client-only or SSR/hybrid contexts
  * It's recommended to not modify this file - please add BYOC imports in corresponding index.*.ts files instead
  */
 
 // Set context properties to be available within BYOC components
-FEAAS.setContextProperties(context);
+FEAAS.setContextProperties({
+  sitecoreEdgeUrl: config.sitecoreEdgeUrl,
+  sitecoreEdgeContextId: config.sitecoreEdgeContextId,
+  siteName: config.sitecoreSiteName,
+  eventsSDK: Events,
```

Comment:
I was discussing that with @yavorsk and I would recommend to leave "sdks" property as it was before. We don't know exactly if we will introduce new sdks in a future, using this naming we will need to make repetitive things and it's better to structure sdks out of common props 'edgeUrl', etc.

Reply:
This approach and type is the Components' team decision, which I'm adjusting our code to. Their current plan is to completely replace the components library with SXA components in short term - so it's unlikely more sdks will be added.

Reply:
I got it. Conceptually this decision doesn't look good